cmd/cli_test.go

```diff
@@ -0,0 +1,120 @@
+// +build !race
```

Comment:
@leodido as discussed privately, we can remove this and temporarily disable the `--race` option in the make file, in order to make this test run on the CI. 
I will do it.